src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1609,13 +1609,7 @@ static void uct_ib_mlx5_devx_check_odp(uct_ib_mlx5_md_t *md,
     struct ibv_mr *mr;
     uint8_t version;
 
-    if (IBV_ACCESS_ON_DEMAND == 0) {
-        reason = "IBV_ACCESS_ON_DEMAND is not supported";
-        goto no_odp;
-    }
-
-    if (!IBV_DEVICE_HAS_ODP(&md->super.dev)) {
-        reason = "device does not support ODP";
```

Comment:
maybe keep this check

Reply:
Why? It is included to `uct_ib_md_check_odp_common`